routes/auth.routes.js

```diff
@@ -0,0 +1,40 @@
+const { Router } = require("express");
+const router = new Router();
+
+const bcryptjs = require("bcryptjs");
+const saltRounds = 10;
+
+const User = require("../models/User.model");
```

Comment:
Nothing to say, but another way to do this type of declaration is to have all the `require` before anything else in the file. Continue doing it like you have does not prevent your code from running, this is mostly a "coding style" tip